lib/exlasticsearch/repo.ex

```diff
@@ -17,6 +17,7 @@ defmodule ExlasticSearch.Repo do
 
   @chunk_size 2000
   @type response :: {:ok, %HTTPoison.Response{}} | {:error, any}
+  @log_level Application.get_env(:exlasticsearch, __MODULE__, []) |> Keyword.get(:log_leve, :debug)
```

Comment:
This won't be configurable post compilation. Is that okay?

Reply:
Yeah I think so.  My guess is people are going to want this set to debug when deployed no matter what and only move it up when testing locally.